src/Compilers/CSharp/Portable/BoundTree/NullabilityRewriter.cs

```diff
@@ -96,15 +192,16 @@ private BoundNode VisitBinaryOperatorBase(BoundBinaryOperatorBase binaryOperator
             bool foundUpdate = false;
             foreach (var originalSymbol in symbols)
             {
-                if (_updatedSymbols.TryGetValue((expr, originalSymbol), out var updatedSymbol))
+                if (originalSymbol is null) continue;
```

Comment:
>if (originalSymbol is null) continue; [](start = 16, length = 37)

Are we going to drop the null from the result?

Reply:
Good catch.

---
In reply to: [329754375](https://github.com/dotnet/roslyn/pull/38769#discussion_r329754375) [](ancestors = 329754375)